apps/web/lib/singleUseSurveys.ts

```diff
@@ -8,20 +8,26 @@ export const generateSurveySingleUseId = (isEncrypted: boolean): string => {
   if (!isEncrypted) {
     return cuid;
   }
-  if (!FORMBRICKS_ENCRYPTION_KEY) {
-    throw new Error("FORMBRICKS_ENCRYPTION_KEY is not defined");
-  }
-  const encryptedCuid = encryptAES128(FORMBRICKS_ENCRYPTION_KEY, cuid);
+
+  const encryptedCuid = symmetricEncrypt(cuid, ENCRYPTION_KEY);
   return encryptedCuid;
 };
 
 // validate the survey single use id
 export const validateSurveySingleUseId = (surveySingleUseId: string): string | undefined => {
-  if (!FORMBRICKS_ENCRYPTION_KEY) {
-    throw new Error("FORMBRICKS_ENCRYPTION_KEY is not defined");
-  }
   try {
-    const decryptedCuid = decryptAES128(FORMBRICKS_ENCRYPTION_KEY!, surveySingleUseId);
+    let decryptedCuid: string | null = null;
+
+    if (surveySingleUseId.length === 64) {
```

Comment:
@pandeymangg Why did you add a check of length === 64 here? Also why did you use AES128 for 64-length string and symmetric decrypt for string of length other than 64?